apps/dashboard/app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -149,7 +149,7 @@ def session_contexts_param
       params.require(:batch_connect_session_context).permit(@session_context.attributes.keys) if params[:batch_connect_session_context].present?
     end
 
-    # Store session context into a cache file
+    # Store session context into a cache file (Returns Pathname)
     def cache_file
```

Comment:
Interestingly enough - I think this is the only method in this change that actually returns something.  Everything else seems to set instance variables. These probably have return values, but they're disregarded and the instance variable is subsequently used. 

That said - I think you want this format that's yarndoc compliant.

```ruby
# Store session context into a cache file
# @return [Pathaname] the cachefile path.
def cache_file
```

Reply:
Fixed in [ad81406](https://github.com/OSC/ondemand/pull/3619/commits/ad8140688e631149a2b890ae98171aca368adc5c)